scripts/scripts.js

```diff
@@ -168,12 +156,6 @@ const loadStyle = (path) => {
   eagerLoad(marquee.querySelector('img'));
 }());
 
-/*
- * ------------------------------------------------------------
- * Edit below at your own risk
- * ------------------------------------------------------------
- */
```

Comment:
Risk is still there, same as for `utils.js`, it might not hurt to keep these for future developers

Reply:
+1